pcsx2/CDVD/CDVD.cpp

```diff
@@ -890,7 +890,9 @@ __fi void cdvdReadInterrupt()
 
 	if (--cdvd.nSectors <= 0)
 	{
-		cdvd.PwOff |= 1<<Irq_CommandComplete;
```

Comment:
it looks like they have both been changed, not sure why we aren't just calling `cdvdSetIrq` though.

There is also another place where we are manually clearing the pwOff, I'm not sure we should be doing that, the BIOS handling for clearing IRQ's should be doing that.